1-js/13-modules/03-modules-dynamic-imports/article.md

```diff
@@ -64,35 +64,35 @@ hi();
 bye();
 ```
 
-Or, if `say.js` has the default export:
+Ou, se `say.js` tiver a exportação padrão>
```

Comment:
Do not translate "default export" to "exportação padrão", as it may cause confusion with the JavaScript keyword. Keep the phrase in English to avoid ambiguity.